src/Subpages/Resources/AssistanceEmployment.jsx

```diff
@@ -0,0 +1,208 @@
+import SecondaryButton from "../../ui-kit/SecondaryButton";
+
+const AssistanceEmployment = () => {
+  const classes = useResourcesStyles();
+  return (
+    <>
+      <Box className={classes.darkBlueBackground}>
+        <Container>
+          <Typography
+            className={classes.headingStyle}
+            variant="h2"
+            align="center"
+          >
+            Laws protect those with convictions from discrimination
+          </Typography>
+          <Typography variant="body1">
+            The Equal Employment Opportunity Commission enforces the most
+            critical law with regards to the protection of felons, as stated in
+            Title VII of the Civil Rights Act of 1964. Felons’ rights may be
+            violated during the search for employment in two ways:
+          </Typography>
+        </Container>
+        <Container className={classes.regularContainer} maxWidth="md">
+          <Grid container>
+            <Grid item xs={12} sm={12} md={4}>
+              <Typography
+                className={classes.headingStyle}
+                variant="h4"
+                align="center"
+              >
+                Adverse Impact
+              </Typography>
+              <Typography variant="body1" align="center">
+                Disparate impact or unintentional discrimination
+              </Typography>
+            </Grid>
+            <Grid item xs={12} sm={12} md={4}>
+              <Typography
+                className={classes.headingStyle}
+                variant="h3"
+                align="center"
+              >
+                OR
+              </Typography>
+            </Grid>
+            <Grid item xs={12} sm={12} md={4}>
+              <Typography
+                className={classes.headingStyle}
+                variant="h4"
+                align="center"
+              >
+                Adverse Treatment
+              </Typography>
+              <Typography variant="body1" align="center">
+                Disparate treatment or intentional discrimination
+              </Typography>
+            </Grid>
+          </Grid>
+        </Container>
+      </Box>
+      <Container className={classes.regularContainer} maxWidth="md">
+        <ListItemMobileSnap
+          text={
+            <div>
+              <Typography className={classes.headingStyle} variant="h4">
+                Want to know if you (dis)qualify for a job?
+              </Typography>
+              <Typography className={classes.contentTextStyle} component="ol">
+                <li>
+                  Click the button to open the Council of State Governments map
+                  in a new tab
+                </li>
+                <li>Select Washington State</li>
+                <li>Search a category under “Employment”</li>
+              </Typography>
+              <SecondaryButton text={"View Map"} linkTo={"https://niccc.csgjusticecenter.org/map/"} />
```

Comment:
This should open in a new tab so as to not force people to leave the site.

Reply:
Good idea.